### PR TITLE
Support Amazon Connect

### PR DESCRIPTION
**Pull request by kvasukib**
*Issue #, if available:*

*Description of changes:*

Add support for Amazon Connect

* Generated SDKs
* Integration test
* Build (Gradle) and release (Maven) support

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: aws-android-sdk-core/src/main/java/com/amazonaws/internal/config/InternalConfig.java [hunk 1]
```diff
@@ -127,6 +127,7 @@ public SignerConfig getSignerConfig(String serviceName, String regionName) {
         //signer uses the servicename 'sagemaker' for Sagemaker Runtime
         ret.put("AmazonSageMakerRuntimeClient", new HttpClientConfig("sagemaker"));
         ret.put("AmazonTextractClient", new HttpClientConfig("textract"));
+        ret.put("AmazonConnectClient", new HttpClientConfig("connect"));
```
Comment (palpatim) [inline aws-android-sdk-core/src/main/java/com/amazonaws/internal/config/InternalConfig.java]: nit: please alphabetize this list

File: settings.gradle [hunk 2]
```diff
@@ -65,3 +65,5 @@ include ':aws-android-sdk-sagemaker-runtime'
 include ':aws-android-sdk-sagemaker-runtime-test'
 include ':aws-android-sdk-textract'
 include ':aws-android-sdk-textract-test'
+include ':aws-android-sdk-connect'
```
Comment (palpatim) [inline settings.gradle]: nit: please alphabetize

File: aws-android-sdk-core/src/main/java/com/amazonaws/internal/config/InternalConfig.java (hunk 1 above)
Comment (kvasukib) [reply aws-android-sdk-core/src/main/java/com/amazonaws/internal/config/InternalConfig.java]: Fixed.

File: settings.gradle (hunk 2 above)
Comment (kvasukib) [reply settings.gradle]: Fixed.

**Pull request merged**